api/python/t4/data_transfer.py

```diff
@@ -17,6 +17,28 @@
 s3_manager = create_transfer_manager(s3_client, TransferConfig())
 
 
+def deserialize_obj(data, target):
```

Comment:
Can you move this back (for now at least)? It won't even work here due to missing imports.

Reply:
I finished the refactor, missing imports should be resolved now